src/components/common/Header/index.tsx

```diff
@@ -26,15 +27,23 @@ type HeaderProps = {
   onBatchToggle?: Dispatch<SetStateAction<boolean>>
 }
 
+function getLogoLink(router: ReturnType<typeof useRouter>): Url {
+  return router.pathname === AppRoutes.home || !router.query.safe
+    ? router.pathname === AppRoutes.welcome.accounts
+      ? AppRoutes.welcome.index
```

Comment:
I don't understand why we are linking to the welcome page. The only thing the user can do there is go to the accounts page again so its an unnecessary click.

Reply:
1. That page is more focused on loging in, so if the person didn't find what they wanted on the accounts page, they probably want to "reset", i.e. go to the "beginning"
2. I think it's better than just staying on the same page (Accounts), alghough we could also make the logo unclickable in that case